src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -50,6 +50,7 @@
         <div
           class="quick-navigation__match-list"
           :class="{ 'active' : processedUserInput.length }"
+          v-bind="{[SCROLL_LOCK_DISABLE_ATTR]: true}"
```

Comment:
Seems to be working for me as-is, but maybe this would need to be moved or added there if #634 gets merged first to match the overflow rules?

Reply:
Since it was working for Marcus, I tested it out again. It seems to be working mostly, but failing inconsistently. I can't seem to figure out exactly how to reproduce it. Could be an issue with the simulator?

Reply:
Ideally with #634 merged, the `data-disable-scroll-lock` will be on the `.quick-navigation__refs` yes :)